website/docs/guides/provider.md

```diff
@@ -0,0 +1,38 @@
+---
+id: provider
+title: Manage providers
+sidebar_position: 1
+---
+### Manage Providers
+
+### Before you start 
+You can manage a provider assigned to a specific project.
+Firstly you need to be registered to the accounting-system and then you need to contact the administrator of the project , that this provider is associated, in order to assign you one or more roles, on the provider.
+Once you gain permissions on the specific provider, you can act on it, according to the permissions. 
+
+### READ the provider
+<details>
+You can display the providers's details. Apply a request to the api. 
+<b> For more details , see <a href="https://argoeu.github.io/argo-accounting/docs/api/provider#get---fetch-an-existing-provider">here</a></b>
+</details>
+
+### ASSIGN installations to the provider
+<details>
+You can assign one or more installations to the installation. Apply a request to the api.
+<b> For more details , see <a href="https://argoeu.github.io/argo-accounting/docs/api/installation#post---create-a-new-installation">here</a></b>
+</details>
+
+### COLLECT all installations assigned to the provider
+<details>
+You can collect and display the details of all the installations assigned to the provider. Apply a request to the api
+<b> For more details , see <a href="https://argoeu.github.io/argo-accounting/docs/api/installation#get-fetch-all-provider-installations">here</a></b>
+</details>
+
+### SEARCH installations
+<details>
+You can search for specific installation assigned to the provider, that matches one or more criteria. Apply a request to the api. You need to provide the search criteria in a specific <b><a href=""> syntax</a></b>. <b> For more details , see <a href="">here</a></b>
+</details>
+
+<b> Also, on the installations assigned to the provider, you are permitted to perform all the actions described at this <a href="../guides/installation"> section</a></b>
```

Comment:
Maybe rephrase to: "You are permitted to perform all the actions described at ....  on the installations assigned to the provider."